src/recordlinker/schemas/pii.py

```diff
@@ -418,23 +432,27 @@ def blocking_keys(self, key: models.BlockingKey) -> set[str]:
 
         if key == models.BlockingKey.BIRTHDATE:
             # NOTE: we could optimize here and remove the dashes from the date
-            vals.update(self.feature_iter(Feature.BIRTHDATE))
-        elif key == models.BlockingKey.MRN:
-            vals.update({x[-4:] for x in self.feature_iter(Feature.MRN)})
+            vals.update(self.feature_iter(Feature(attribute=FeatureAttribute.BIRTHDATE)))
+        elif key == models.BlockingKey.IDENTIFIER:
+            vals.update({
+                f"{type_part}:{authority_part}:{value_part[-4:]}"
```

Comment:
```suggestion
                f"{type_part}:{authority_part[:2]}:{value_part[-4:]}"
```

https://github.com/CDCgov/RecordLinker/pull/125/files#diff-565877605f59af85f1d668e71eecf89e9a43942b2d31bea13adb6ac9f30aeb43R154-R158